keymap/vim.js

```diff
@@ -4308,7 +4308,7 @@
               if (decimal + hex + octal > 1) { return 'Invalid arguments'; }
               number = decimal && 'decimal' || hex && 'hex' || octal && 'octal';
             }
-            if (args.eatSpace() && args.match(/\/.*\//)) { 'patterns not supported'; }
```

Comment:
Note that `args.eatSpace()` can never return `true` here, since there is another `eatSpace()` just a few lines above.